packages/app-project/src/shared/components/WorkflowSelector/components/WorkflowSelectButtons/WorkflowSelectButtons.spec.js

```diff
@@ -45,19 +45,19 @@ describe('Component > WorkflowSelector > WorkflowSelectorButtons', function () {
     it('should render a workflow link for each workflow', function () {
       const { getAllByRole } = render(
         <RouterContext.Provider value={mockRouter}>
-          <WorkflowSelectButtons onSelect={() => { }} workflows={workflows} />
+          <WorkflowSelectButtons workflows={workflows} />
         </RouterContext.Provider>
       )
       expect(getAllByRole('link')).to.have.lengthOf(workflows.length)
     })
   })
 
-  describe('when workflow assignment is enabled', function () {
+  describe.skip('when workflow assignment is enabled', function () {
```

Comment:
Noting that this block is currently skipped. I'll look into refactoring the unit tests a little bit.